<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionCommande</class>
 <widget class="QMainWindow" name="GestionCommande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="StackedWidgetCommande">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>720</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="CRUD_1">
     <widget class="QLineEdit" name="BarreRechercheCommande">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>681</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="ListeCommande">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>1001</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QPushButton" name="RechercheCommande">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>230</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Statistique">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>620</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>STATISTIQUE</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="BG_CRUD_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1080</width>
        <height>720</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Graphique Qt/Sans titre - 1.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="Imprimer">
      <property name="geometry">
       <rect>
        <x>544</x>
        <y>620</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IMPRIMER</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="TriButton">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>230</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TRI</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="TriNom">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>260</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(245, 128, 32);</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="TriDate">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>230</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(245, 128, 32);</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <zorder>BG_CRUD_1</zorder>
     <zorder>BarreRechercheCommande</zorder>
     <zorder>ListeCommande</zorder>
     <zorder>RechercheCommande</zorder>
     <zorder>Statistique</zorder>
     <zorder>Imprimer</zorder>
     <zorder>TriButton</zorder>
     <zorder>TriNom</zorder>
     <zorder>TriDate</zorder>
    </widget>
    <widget class="QWidget" name="CRUD_2">
     <widget class="QPushButton" name="DeleteButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>560</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>560</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UPDATE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CinClientCRUD">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>320</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="ReferenceProduitCRUD">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="BG_CRUD_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1080</width>
        <height>720</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Graphique Qt/Sans titre - 1.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="Test">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>480</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Back">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>560</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NumeroCRUD">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>BG_CRUD_2</zorder>
     <zorder>DeleteButton</zorder>
     <zorder>UpdateButton</zorder>
     <zorder>CinClientCRUD</zorder>
     <zorder>ReferenceProduitCRUD</zorder>
     <zorder>Test</zorder>
     <zorder>Back</zorder>
     <zorder>NumeroCRUD</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
